components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -245,6 +248,9 @@ func (g *Graphqlizer) GardenerUpgradeInputToGraphQL(in gqlschema.GardenerUpgrade
       {{- end }}
       {{- if .MaxUnavailable }}
       maxUnavailable: {{ .MaxUnavailable }},
+      {{- end }}
+	  {{- if .ExposureClassName }}
+	  exposureClassName: "{{ .ExposureClassName }}",
```

Comment:
nit: is the indentation correct? just wondering if it's not supposed to be the same indent as `maxUnavailable` like it is few lines above

Reply:
Yeah, I saw this already, it appears to be a GH Display issue with tabs/spaces. The indentationa already looks odd on [main](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/provisioner/graphqlizer.go#L95).

However, I see if I can switch to spaces only. Thanks for pointing this out again!